src/main/kotlin/org/rust/lang/core/types/BoundElement.kt

```diff
@@ -52,7 +52,7 @@ data class BoundElement<out E : RsElement>(
         )
 
     override fun superVisitWith(visitor: TypeVisitor): Boolean = assoc.values.any { visitor.visitTy(it) } ||
```

Comment:
Shouldn't we use `visitWith` instead of `visitTy` and `visitRegion` here?

Reply:
Fixed